rasa/nlu/emulators/wit.py

```diff
@@ -1,32 +1,52 @@
+from collections import defaultdict
 from typing import Any, Dict, List, Text
 
 from rasa.nlu.emulators.no_emulator import NoEmulator
+from rasa.shared.nlu.constants import (
+    ENTITIES,
+    ENTITY_ATTRIBUTE_TYPE,
+    ENTITY_ATTRIBUTE_ROLE,
+    ENTITY_ATTRIBUTE_VALUE,
+    ENTITY_ATTRIBUTE_END,
+    ENTITY_ATTRIBUTE_START,
+    TEXT,
+    INTENT,
+)
 
 
 class WitEmulator(NoEmulator):
-    def __init__(self) -> None:
+    """Emulates the response format of this wit.ai endpoint:
+    https://wit.ai/docs/http/20200513/#get__message_link
+    """
 
+    def __init__(self) -> None:
         super().__init__()
         self.name = "wit"
 
-    def normalise_response_json(self, data: Dict[Text, Any]) -> List[Dict[Text, Any]]:
+    def normalise_response_json(self, data: Dict[Text, Any]) -> Dict[Text, Any]:
         """Transform data to wit.ai format."""
 
```

Comment:
should we use a complete docstring?

Reply:
Agree but I would say the method we're overriding (the one from `NoEmulator`) needs to be documented instead. Will do that!